src/components/InvoicesTable.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import moment from 'moment';
+
+export default class InvoicesTable extends Component {
+  render() {
+    const invoiceRows = this.props.invoices
+                            .valueSeq()
+                            .sortBy(invoice => -invoice.get('id'));
```

Comment:
Try to use a comparator to sort in descending order. If that is not possible, you could just `reverse` the sequence.